pkgs/os-specific/linux/intel-sgx/default.nix

```diff
@@ -0,0 +1,286 @@
+      cp ${build}/libsample_libcrypto.so $out/SampleCode/RemoteAttestation/sample_libcrypto/
+      cp sdk/sample_libcrypto/sample_libcrypto.h $out/SampleCode/RemoteAttestation/sample_libcrypto/
+
+      # copy libraries
+      mkdir -p $out/lib
+      cp ${build}/libsgx_ptrace.so $out/lib/
+      cp ${build}/libsgx_tcrypto.a $out/lib/
+      cp ${build}/libsgx_tkey_exchange.a $out/lib/
+      cp ${build}/libsgx_trts.a $out/lib/
+      cp ${build}/libsgx_trts_sim.a $out/lib/
+      cp ${build}/libsgx_tservice.a $out/lib/
+      cp ${build}/libsgx_tservice_sim.a $out/lib/
+      cp ${build}/libsgx_tstdc.a $out/lib/
+      cp ${build}/libsgx_tcxx.a $out/lib/
+      cp ${build}/libsgx_tcmalloc.a $out/lib/
+      cp ${build}/libsgx_tswitchless.a $out/lib/
+      cp ${build}/libsgx_uswitchless.a $out/lib/
+      cp ${build}/libsgx_ukey_exchange.a $out/lib/
+      cp ${build}/libsgx_capable.a $out/lib/
+      cp ${build}/libsgx_capable.so $out/lib/
+      cp ${build}/libsgx_uprotected_fs.a $out/lib/
+      cp ${build}/libsgx_tprotected_fs.a $out/lib/
+      cp ${build}/libsgx_pcl.a $out/lib/
+      cp ${build}/libsgx_pclsim.a $out/lib/
+      cp ${build}/libsgx_pthread.a $out/lib/
+      cp ${build}/libsgx_omp.a $out/lib/
+
+      cp ${build}/libsgx_epid_deploy.so $out/lib/libsgx_epid.so
+      cp ${build}/libsgx_epid_sim.so $out/lib/
+      cp ${build}/libsgx_launch_deploy.so $out/lib/libsgx_launch.so
+      cp ${build}/libsgx_launch_sim.so $out/lib/
+      cp ${build}/libsgx_quote_ex_deploy.so $out/lib/libsgx_quote_ex.so
+      cp ${build}/libsgx_quote_ex_sim.so $out/lib/
+      cp ${build}/libsgx_uae_service_deploy.so $out/lib/libsgx_uae_service.so
+      cp ${build}/libsgx_uae_service_sim.so $out/lib/
+      cp ${build}/libsgx_urts_deploy.so $out/lib/libsgx_urts.so
+      cp ${build}/libsgx_urts_sim.so $out/lib/libsgx_urts_sim.so
+
+      cp -r ${build}/*.a $out/lib/
+      cp -r ${build}/*.so $out/lib/
+
+      cp -r ${build}/gdb-sgx-plugin $out/lib/
+      cp external/dcap_source/QuoteGeneration/build/linux/libsgx_dcap_tvl.a $out/lib/
+
+      mkdir -p $out/lib/cve_2020_0551_cf
+      cp -r build/linuxCF/*.a $out/lib/cve_2020_0551_cf/
+      cp external/dcap_source/QuoteGeneration/build/linuxCF/libsgx_dcap_tvl.a $out/lib/cve_2020_0551_cf/
+
+      mkdir -p $out/lib/cve_2020_0551_load
+      cp -r build/linuxLOAD/*.a $out/lib/cve_2020_0551_load/
+      cp external/dcap_source/QuoteGeneration/build/linuxLOAD/libsgx_dcap_tvl.a $out/lib/cve_2020_0551_load/
+
+      # copy binaries
+      mkdir -p $out/bin
+      cp ${build}/sgx_config_cpusvn $out/bin/
+      cp ${build}/sgx_edger8r $out/bin/
+      cp ${build}/sgx_sign $out/bin/
+      cp ${build}/sgx_encrypt $out/bin/
+      cp ${build}/gdb-sgx-plugin/sgx-gdb $out/bin/
+
+      # link sdk folder (several tools expect this exact folder structure)
+      mkdir -p $out/sdk
+      cp common/buildenv.mk $out/sdk/
+
+      ln -s $out/include $out/sdk/include
+      ln -s $out/lib $out/sdk/lib
```

Comment:
Can the sdk directory be moved to follow FHS?

Reply:
Yep, I've moved this to `/usr/share/sgxsdk` now so it's the same as the Debian package.